### ISSUE TITLE
TFLite low detection accuracy

### ISSUE THREAD

**Issue by patrick-ucr** [tensorflow/models#6477]
### System information
- **What is the top-level directory of the model you are using**: tensorflow/tensorflow/models
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04.6
- **TensorFlow installed from (source or binary)**: Source
- **TensorFlow version (use command below)**: 1.12.0
- **Bazel version (if compiling from source)**: 0.19.1
- **CUDA/cuDNN version**: CUDA 10.0 
- **GPU model and memory**: Tesla P100 12GB Memory
- **Exact command to reproduce**:
`bazel run -c opt tensorflow/lite/toco:toco -- --input_file=tflite_graph.pb --output_file=ssdlite_mobilenet_v2_float.tflite --input_format=TENSORFLOW_GRAPHDEF --output_format=TFLITE input_shapes=1,300,300,3 --input_arrays=normalized_input_image_tensor --output_arrays='TFLite_Detection_PostProcess','TFLite_Detection_PostProcess:1','TFLite_Detection_PostProcess:2','TFLite_Detection_PostProcess:3' --inference_type=FLOAT --allow_custom_ops`

### Describe the problem
I trained a custom model from pretrained ssdlite_mobilenet_v2_coco from the model zoo. When I export it using export_inference_graph.py and test it on Android demo (tensorflow/tensorflow/examples/android/) detection accuracy was good (I use test video datasets).

However when I export it with export_tflite_ssd_graph.py and the above toco script to have tflite FLOAT model file without a serious error and use it in TFLite Android demo (tensorflow/tensorflow/lite/examples/android), following this instruction
[https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/running_on_mobile_tensorflowlite.md](url)

The problem is that now detection accuracy is very bad (recall is fine but precision is not) because of many false positive detections. There is another way of conversion by using tflite_converter but not sure how to do it for a detector model (see some classifier examples)

I plan to convert this to a quantized model but get stuck at this point and after looking at tensorflow issues, I can't see a similar problem experienced by others.

Here is a sample screenshot.
[https://imgur.com/a/pEHVxnm](url)

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
